packages/devtools_app/lib/src/performance/performance_model.dart

```diff
@@ -293,14 +293,14 @@ class OfflinePerformanceData extends PerformanceData {
                 .cast<String, dynamic>())
         : null;
 
-    final double displayRefreshRate =
+    final num displayRefreshRate =
```

Comment:
rather than making this a num prefer to convert the json to a double. generally use num only where really necessary as it is easier to just reason about data consistently being int or double.

Reply:
done